core/templates/dev/head/components/question-directives/questions-list/questions-list.directive.ts

```diff
@@ -190,25 +190,52 @@ oppia.directive('questionsList', [
               controller: [
                 '$scope', '$uibModalInstance',
                 function($scope, $uibModalInstance) {
-                  $scope.selectedSkillIds = [];
+                  var fillSkillDifficulties = function(length) {
+                    $scope.selectedSkillDifficulties = [];
+                    for (var i = 0; i < length; i++) {
+                      $scope.selectedSkillDifficulties.push('0.3');
+                    }
+                  };
+                  $scope.selectedSkillIds = linkedSkillIds;
+                  $scope.selectedSkillDescriptions = [];
+                  fillSkillDifficulties(ctrl.newQuestionSkillIds.length);
+                  $scope.selectDifficulty = selectDifficulty;
```

Comment:
If this is a boolean, rename to not start with a verb.

Reply:
Done